.gitignore

```diff
@@ -1,9 +0,0 @@
-build_files/
-build/
-
-arm_*.elf
-arm_*.bin
-arm_*_bin.h
-*.map
-
-iosu_payload/
```

Comment:
Would be good to keep a gitignore, if there are conflicts you can add or remove them as you need